src/array_api_typing/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Static typing support for the array API standard."""
+
+__all__: list[str] = []
```

Comment:
alternative workaround for type-checkers that still don't realize that `__all__` is always a sequence of `str` in 2024:

```suggestion
__all__ = ()
```